File: 2847-find-maximum-number-of-string-pairs/find-maximum-number-of-string-pairs.py
class Solution:
    def maximumNumberOfStringPairs(self, words: List[str]) -> int:
        count=0
        for i in range(0,len(words)):
            for j in range(i+1,len(words)):
                if words[i]==words[j][::-1]:
                    count+=1
        return count

File: 2847-find-maximum-number-of-string-pairs/find-maximum-number-of-string-pairs.java
class Solution {
    public int maximumNumberOfStringPairs(String[] words) {
        int count=0;
        StringBuilder r=new StringBuilder();
        for(int i=0;i<words.length;i++){
            for(int j=i+1;j<words.length;j++){
                r.setLength(0);
                r.append(words[j]);
                if(words[i].equals(r.reverse().toString())){
                    count+=1;
                }
            }
        }
        return count;
    }
}